src/views/CharacterListView.js

```diff
@@ -3,19 +3,20 @@ import { connect } from "react-redux";
 
 import { CharacterList } from "../components";
 // import actions
+import { fetchDataStart } from "../actions";
 
 class CharacterListView extends React.Component {
-  constructor() {
-    super();
-  }
-
   componentDidMount() {
     // call our action
+    this.props.fetchDataStart();
   }
 
   render() {
+    console.log(this.props.fetching);
+    console.log(this.props.characters);
     if (this.props.fetching) {
       // return something here to indicate that you are fetching data
+      return <p>Loading...</p>;
     }
```

Comment:
```suggestion
    }
    else if (this.props.error) {
      return <p>{this.props.error.message}</p>
    }
```
I would also add some error handling here